packages/react-native-builder-bob/src/index.ts

```diff
@@ -584,6 +585,15 @@ async function buildTarget(
         report,
       });
       break;
+    case 'custom':
+      await customTarget({
+        options: targetOptions,
+        source: path.resolve(root, source),
+        output: path.resolve(root, output, 'typescript'),
```

Comment:
TypeScript? 🤔

Reply:
They said copying and pasting always works 🙈